clusterloader2/pkg/prometheus/manifests/prometheus-prometheus.yaml

```diff
@@ -6,15 +6,36 @@ apiVersion: monitoring.coreos.com/v1
 kind: Prometheus
```

Comment:
It looks like this file drops overrides some existing configurations used by our tests. I added few comments, but I think there are also other cases that should be rolled back.